elliott/elliottlib/cli/find_bugs_golang_cli.py

```diff
@@ -457,12 +457,12 @@ def compare(b1, b2):
 
             if fixed:
                 if self.update_tracker:
-                    if not self.art_jira:
-                        raise ElliottFatalError("Please provide ART Jira ticket for reference with --art-jira")
-                    message = f"Refer to {self.art_jira} for details"
+                    message = ''
+                    if self.art_jira:
+                        message = f"Refer to {self.art_jira} for details."
                     comment = f"{comment} {message}"
-                    if bug.status in ['New', 'ASSIGNED', 'POST']:
-                        self.jira_tracker.update_bug_status(bug, 'MODIFIED', comment=comment, noop=self.dry_run)
+                    if bug.status in ['New', 'ASSIGNED', 'POST', 'MODIFIED']:
+                        self.jira_tracker.update_bug_status(bug, 'ON_QA', comment=comment, noop=self.dry_run)
```

Comment:
I had set this to Modified out of caution. I was relying on the "normal" bug sweep to then move these from modified -> on_qa. Now that we are running them together, it's better to move to on_qa directly and not have this dependency and confusion.